### ISSUE TITLE
Add UI for rightToLeft property

### ISSUE THREAD

**Issue by miyconst** [Juicy/juicy-tile-editor#77]
There is a new property `rightToLeft` (https://github.com/Juicy/juicy-tile-list/issues/38), but `juicy-tile-editor` lacks UI for it.
- [ ] Add UI for `rightToLeft` property.
- [ ] Add titles with explanation to direction buttons.

**Comment by tomalec** [comment]
Maybe we could also change icons to be more vertical/horizontal, and do not suggest left to right?

**Comment by miyconst** [comment]
Okay, I will also look for some better icons.

**Comment by miyconst** [comment]
Here are four options to choose from:

![2015-07-21](https://cloud.githubusercontent.com/assets/6435556/8798523/8f8b6dba-2fab-11e5-87c7-9d8ec94ac2e8.png)
![2015-07-21 1](https://cloud.githubusercontent.com/assets/6435556/8798524/8f8e6f88-2fab-11e5-8b60-bd53faa623a0.png)
![2015-07-21 2](https://cloud.githubusercontent.com/assets/6435556/8798525/8fa90244-2fab-11e5-9c9d-8e9635cc1de4.png)
![2015-07-21 3](https://cloud.githubusercontent.com/assets/6435556/8798526/8fad0150-2fab-11e5-98a0-a2f7bc82a740.png)

I like the latest one, but it uses css `transform: rotate(90deg);` for the vertical icon.

**Comment by warpech** [comment]
I like the last one as well

**Comment by tomalec** [comment]
I would say that forth is perfect :)

**Comment by miyconst** [comment]
Now it is getting cool:

![image](https://cloud.githubusercontent.com/assets/6435556/8800404/15318c1a-2fbb-11e5-8763-9c8706cc50bb.png)

The `Direction` buttons change icon alignment according to the `Horizontal` & `Vertical` arrows.

The only question is why the property `bottomUp` is named so?
I would go with one of these:
- `rightLeft` & `downUp` 
- `rightToLeft` & `bottomToTop` - this one I like more

**Comment by tomalec** [comment]
I like the new buttons.

Regards naming, what about `upwards`(, `downwards`), `leftward`(, `rightward`).

I think it's not that bad that they are not similar to each other currently, as they are not the part of the same set. You are not choosing one from: `leftToRight`, `rightToLeft`, `bottomToTop`, `topToBottom`, `sideToSide`, `levelToLevel` (?).

`rightToLeft` is completely independent choice from `bottomUp`.

**Comment by tomalec** [comment]
Unless we will make it a choice of 1 of 8 directions:
- downwardsLeftToRight
- rightwardsTopToBottom
- leftwardsTopToBottom
- ...

but I don't think it would be usable.

**Comment by miyconst** [comment]
> Regards naming, what about `upwards`(, `downwards`), `leftward`(, `rightward`).

+1

> `rightToLeft` is completely independent choice from `bottomUp`

Yes, they are independent, just for me it would be easier to remember properties with same naming convention. Actually, I created some examples with `bottomToTop: true`, just replacing directions in `rightToLeft` property. The examples didn't work, obviously. Then I found the real property name.

The `borderColor` and `backgroundColor` are also two independent properties, but we don't name it `borderColor` and `backgroundHue`.

Anyway, this is not something important, if you like the naming, I am okay with it.
